### ISSUE TITLE
Install not working on OSX Node12

### ISSUE THREAD

**Issue by JasperFioole21** [song940/node-bluetooth#38]
```
➜  bluetooth node-gyp build                          
gyp info it worked if it ends with ok
gyp info using node-gyp@5.0.3
gyp info using node@12.6.0 | darwin | x64
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
make: *** No rule to make target `Release/obj.target/BluetoothSerialPort/src/osx/DeviceINQ.o', needed by `Release/BluetoothSerialPort.node'.  Stop.
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/node-gyp/lib/build.js:196:23)
gyp ERR! stack     at ChildProcess.emit (events.js:203:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 18.6.0
gyp ERR! command "/usr/local/Cellar/node/12.6.0/bin/node" "/usr/local/bin/node-gyp" "build"
gyp ERR! cwd /Development/Node/bluetooth
gyp ERR! node -v v12.6.0
gyp ERR! node-gyp -v v5.0.3
gyp ERR! not ok 
➜  bluetooth 
```

**Comment by ekoeryanto** [comment]
I confirm that node 12 is not working to on windows

**Comment by extempl** [comment]
The downgrade doesn't fix the issue #48
